include/mbedtls/config.h

```diff
@@ -2728,6 +2728,27 @@
  */
 #define MBEDTLS_ERROR_C
 
+/**
+ * \def MBEDTLS_CRC_C
+ *
+ * Enable the CRC calculating module
+ *
+ * Module:  library/crc.c
+ *
+ * This module enables mbedtls_crc_update.
+ */
+//#define MBEDTLS_CRC_C
+
+/**
+ * \def MBEDTLS_VALIDATE_AES_KEYS
+ *
+ * Enable validation of AES keys by checking their CRC
+ * after every encryption/decryption.
+ *
+ * Module:  library/aes.c
+ *
+ */
+//#define MBEDTLS_VALIDATE_AES_KEYS
```

Comment:
@AndrzejKurek , let's name  the MBEDTLS_VALIDATE_AES_KEYS_INEGRITY, to make the purpose more clear

Reply:
Done.